### PR TITLE
[script.module.tmdbsimple] 2.2.0+matrix.1

### PR DESCRIPTION
**Pull request by rmrector**
### Description
Matrix version of existing Leia module. For the imminent Matrix version of the TMDB python scraper.

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

### CODE REVIEW

**Comment by rmrector** [comment]
Matrix version of the TMDB scraper: https://github.com/xbmc/repo-scrapers/pull/202

File: script.module.tmdbsimple/addon.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    <import addon="xbmc.python" version="3.0.0"/>
+    <import addon="script.module.requests" version="2.15.1"/>
+  </requires>
+  <extension point="xbmc.python.module"
+             library="lib" />
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en_GB">A wrapper for The Movie Database API v3</summary>
+    <description lang="en_GB">Packed for Kodi from https://github.com/celiao/tmdbsimple</description>
+    <platform>all</platform>
+    <license>GPL-3.0</license>
```
Comment (enen92) [inline script.module.tmdbsimple/addon.xml:16]: GPL-3.0-only or GPL-3.0-or-later (depending on the license you bundle)

File: script.module.tmdbsimple/addon.xml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    <website>https://github.com/rmrector/script.module.tmdbsimple</website>
```
Comment (enen92) [inline script.module.tmdbsimple/addon.xml:17]: website here looks like tmdb and not exactly your repo

Comment (rmrector) [reply script.module.tmdbsimple/addon.xml:17]: https://github.com/xbmc/repo-scripts/pull/1251#discussion_r360692039

What exactly do you want changed, and to what exactly?

Comment (enen92) [reply script.module.tmdbsimple/addon.xml:17]: Disregard, it's right

File: script.module.tmdbsimple/addon.xml (hunk 1 above)
Comment (enen92) [reply script.module.tmdbsimple/addon.xml:16]: Do you want to adjust the license identifier as part of this PR or do you want to adjust it in a future update?

Comment (rmrector) [reply script.module.tmdbsimple/addon.xml:16]: Later works for me.

**Pull request merged**